### PR TITLE
DOC-3035: SFD - reflect new registration/sign in mobile workflows

### PR DESCRIPTION
**Pull request by catong**
## [DOC-3035](https://openedx.atlassian.net/browse/DOC-3035)

This PR updates student-facing "getting started" doc so that it mentions the mobile apps and reflects the new ability in the mobile apps to browse courses before having to either sign in or register. (MA-2377, MA-2378)
### Date Needed (optional)

If the release date of a feature is known or estimated, provide it to give reviewers guidance on turnaround time.
### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 
- [x] Subject matter expert: @mikekatz 
- [x] Subject matter expert: @bguertin
- [x] Doc team review (sanity check/copy edit/dev edit): @lamagnifica 
- [x] Product review: @marcotuts 

FYI: @BenjiLee 
### Testing
- [x] Ran ./run_tests.sh without warnings or errors
### HTML Version (optional)
- [ ] Build an RTD draft for your branch and add a link here
### Post-review
- [ ] Add a comment with the description of this change or link this PR to the next release notes task.
- [x] Squash commits

### CODE REVIEW

**Comment by mikekatz** [comment]
@catong 👍 looks good, but I do not know when this will land for iOS.

**Comment by ghost** [comment]
:+1: Looks good

**Comment by catong** [comment]
@lamagnifica Finished moving includes into the main file. Ready for your review, thanks.
I did not create an RTD project but it should be relatively easy to get this branch and build "students" and "open_edx_students" locally to compare conditions.

File: en_us/shared/getting_started/create_activate_account.rst [hunk 1]
```diff
@@ -26,20 +26,21 @@ Creating an account for edx.org or edX Edge has two steps.
 Create an Account
 ==================
 
-You create, or register, an edx.org or edX Edge account on the account
-registration page. For edx.org, you can also register from the edX mobile app.
+You create, or register, an edX or edX Edge account on the account
+registration page. You can also register edX accounts on the edX mobile app.
```
Comment (lamagnifica) [inline en_us/shared/getting_started/create_activate_account.rst]: Last sentence has an odd switch to the plural. Maybe,
You can also use the edX mobile app to register your account. (?)

File: en_us/shared/getting_started/create_activate_account.rst [hunk 2]
```diff
@@ -26,20 +26,21 @@ Creating an account for edx.org or edX Edge has two steps.
...
-  When you register an account, you specify a username that will represent you
-  throughout the edx.org or edX Edge site. Course team members and other
-  learners see only your username. You cannot change your username after you
-  create your account. EdX recommends that you select your username carefully.
+.. note::   When you register an account, you specify a username that will
+   represent you on edX or on edX Edge. Your username, not your full name, is
+   used to identify you to course team members and other learners. Your full
+   name is used on any certificates that you earn. You cannot change your public
+   username after you register. EdX recommends that you select your username
+   carefully.
```
Comment (lamagnifica) [inline en_us/shared/getting_started/create_activate_account.rst:37]: maybe move this earlier, and take it out of the note format?
(I know you didn't change anything, it just always strikes me odd when a prerequisite that requires some thought isn't part of the conceptual info, but right before the steps begin.)

File: en_us/shared/getting_started/create_activate_account.rst [hunk 3]
```diff
@@ -26,20 +26,21 @@ Creating an account for edx.org or edX Edge has two steps.
...
 
-#. Go to the edx.org or edX Edge registration page.
+#. Go to the edX or edX Edge registration page.
 
-   * For edx.org, go to https://courses.edx.org/register or open the edX
+   * For edX, go to https://courses.edx.org/register or open the edX
```
Comment (lamagnifica) [inline en_us/shared/getting_started/create_activate_account.rst]: comma after the URL?

File: en_us/shared/getting_started/create_activate_account.rst [hunk 4]
```diff
@@ -60,7 +61,7 @@ To register an account on edx.org or edX Edge, follow these steps.
    information can help course teams better understand their learners and
    improve their courses.
 
-#. Select the check box to accept the edX terms of service and honor code.
+#. Select the option to accept the edX terms of service and honor code.
```
Comment (lamagnifica) [inline en_us/shared/getting_started/create_activate_account.rst]: Maybe just start the sentence at "Accept"?

File: en_us/shared/students/SFD_mobile.rst [hunk 5]
```diff
@@ -26,27 +26,157 @@ course discussions, and get started on homework and other assignments. To
 complete an entire course, you use a web browser on a computer.
 
 
-.. The following general Q&A applies to the edX mobile app for partners/edx.org only
-.. Alison, DOC-1840, June 2015
+.. The "General Questions" and "Getting Started" sections that follow apply only to edX partners/edx.org only
 
 .. only:: Partners
```
Comment (lamagnifica) [inline en_us/shared/students/SFD_mobile.rst:32]: it looks like maybe the indentation is missing? hard to tell in GitHub though

File: en_us/shared/students/SFD_mobile.rst [hunk 6]
```diff
@@ -26,27 +26,157 @@ course discussions, and get started on homework and other assignments. To
...
+
+
+ ==================================================
+ How do I enroll in a course?
+ ==================================================
 
-    *************************
-    Getting Started
-    *************************
+ When you have found a course that you want to take, select the course to view
```
Comment (lamagnifica) [inline en_us/shared/students/SFD_mobile.rst]: After you find... (?)

File: en_us/shared/students/SFD_mobile.rst [hunk 7]
```diff
@@ -58,11 +188,56 @@ Course Questions
 Can I take a course entirely on my mobile device?
 ========================================================
 
-Not entirely. With the edX mobile app, you can download course videos to watch
-when you do not have an Internet connection. When you have an Internet
-connection, you can also read course announcements and content, participate in
-course discussions, and do some, but not all, of the problems in your
-assignments. To complete an entire course, you use a web browser on a computer.
+Not entirely. You can download course videos to watch in the mobile app when you
+do not have an Internet connection. When you have an Internet connection, you
```
Comment (lamagnifica) [inline en_us/shared/students/SFD_mobile.rst:192]: is Internet lc or icaps?

File: en_us/shared/students/SFD_mobile.rst [hunk 8]
```diff
@@ -86,15 +261,7 @@ assignments. To complete an entire course, you use a web browser on a computer.
...
-========================================================
-How do I post questions on the discussion board?
-========================================================
-
-You can read and contribute to course discussions in the edX mobile apps
-whenever you have an Internet connection. In your course, select **Discussion**.
-You can then browse through different topics or search for words or phrases that
-interest you. For information about course discussions, see :ref:`Course
-Discussions`.
+..Video Questions section is for both partners and Open edX
```
Comment (lamagnifica) [inline en_us/shared/students/SFD_mobile.rst]: need a space after the ..

File: en_us/shared/students/SFD_mobile.rst [hunk 9]
```diff
@@ -123,11 +290,6 @@ To download videos, you must be connected to the Internet. Select the
 
 * If you select download for a video, you download only that video.
 
-.. The following image is an early version of what will be available eventually for everyone; currently only for Open edX (adds notifications, assessments, discussions)
-.. Alison, DOC-1840, June 2015
-
-.. only:: Open_edX
-
   This example lists subsections in a course and shows how many videos
   that will download when you select the icon.
```
Comment (lamagnifica) [inline en_us/shared/students/SFD_mobile.rst]: strike "that"? this reads oddly...

File: en_us/students/source/SFD_account.rst [hunk 10]
```diff
@@ -18,9 +18,10 @@ Introducing edx.org and edX Edge
 EdX hosts courses on the `edx.org`_ and `edX Edge`_ websites.
 
 * The edx.org website has massive open online courses (MOOCs) that are designed
-  and led by edX partner organizations. This website lists all available
-  courses publicly, and learners from around the world can find and enroll in
-  these courses.
+  and led by edX partner organizations. This website lists all available courses
+  publicly, and learners from around the world can find and enroll in these
+  courses. When we refer to "edX" accounts and courses we are referring to
+  accounts and courses on edx.org.
```
Comment (lamagnifica) [inline en_us/students/source/SFD_account.rst:24]: awesome. Not sure if you want to search the rest of the learner's guide for use of "edx.org" and replace with edX where applicable?

**Comment by lamagnifica** [comment]
cool. 👍 though looking at it, I remember why I set up the separate sections -- every line needed to be indented for the .. only:: to work...

File: en_us/shared/students/SFD_mobile.rst (hunk 5 above)
Comment (catong) [inline en_us/shared/students/SFD_mobile.rst:32]: Nope, not missing. Indented by a single space which is difficult to see in Github. Tested and works!

File: en_us/shared/students/SFD_mobile.rst (hunk 7 above)
Comment (catong) [inline en_us/shared/students/SFD_mobile.rst:192]: Agh. I want it to be lc. Will make a proposal to the team and see if we can change our style :-)

File: en_us/students/source/SFD_account.rst (hunk 10 above)
Comment (catong) [inline en_us/students/source/SFD_account.rst:24]: I'll create a separate story :-)

**Comment by marcotuts** [comment]
Looks great! Thanks! 👍

**Comment by catong** [comment]
Merging https://github.com/edx/edx-documentation/pull/1075

**Pull request merged**